src/renderers/WebGL3DRenderTarget.js

```diff
@@ -0,0 +1,33 @@
+import { WebGLRenderTarget } from './WebGLRenderTarget.js';
+import { Data3DTexture } from '../textures/Data3DTexture.js';
+
+class WebGL3DRenderTarget extends WebGLRenderTarget {
+
+	constructor( width, height, depth, options = {} ) {
+
+		super( width, height, options );
+
+		this.depth = depth;
+
+		this.texture = new Data3DTexture( null, width, height, depth );
+
+		if ( options.format !== undefined ) this.texture.format = options.format;
+		if ( options.type !== undefined ) this.texture.type = options.type;
+		if ( options.mapping !== undefined ) this.texture.mapping = options.mapping;
+		if ( options.wrapS !== undefined ) this.texture.wrapS = options.wrapS;
+		if ( options.wrapT !== undefined ) this.texture.wrapT = options.wrapT;
+		if ( options.wrapR !== undefined ) this.texture.wrapR = options.wrapR;
+		if ( options.magFilter !== undefined ) this.texture.magFilter = options.magFilter;
+		if ( options.minFilter !== undefined ) this.texture.minFilter = options.minFilter;
+		if ( options.anisotropy !== undefined ) this.texture.anisotropy = options.anisotropy;
+		if ( options.encoding !== undefined ) this.texture.encoding = options.encoding;
```

Comment:
@Mugen87 Following the conversation from above...

I don't think we should continue the `options` parameter pattern either...

This code tends to be a bit annoying to deal with:

```js
const renderTarget = new THREE.WebGLArrayRenderTarget( width, height, depth, {
	format: THREE.RedFormat,
	type: THREE.UnsignedByteType
} );
```

I think this is better (plus, it teaches the dev that there's a texture inside the rendertarget):

```js
const renderTarget = new THREE.WebGLArrayRenderTarget( width, height, depth );
renderTarget.texture.format = THREE.RedFormat;
renderTarget.texture.type = THREE.UnsignedByteType;
```

And, most importantly, keeps our internal code simpler.

Reply:
Okay, let's give this a try.